date/date_c.php

```diff
@@ -344,7 +344,7 @@ public function sub(DateInterval $interval): DateTimeImmutable {}
      * @link https://secure.php.net/manual/en/datetime.diff.php
      * @param DateTimeInterface $targetObject <p>The date to compare to.</p>
      * @param bool $absolute [optional] <p>Should the interval be forced to be positive?</p>
-     * @return DateInterval
+     * @return DateInterval|false
```

Comment:
Please document why it can be false

Reply:
@williamdes https://www.php.net/manual/en/datetime.diff.php#refsect1-datetime.diff-returnvalues